cvm-attestation/src/ImdsClient.py

```diff
@@ -103,6 +103,7 @@ def get_vcek_certificate(self):
         if response.status_code == 200:
           self.log.info("Received certificate successfully")
           data_json = json.loads(response.text)
+          self.log.info(f"Reported TCB: {data_json['tcbm']}")
```

Comment:
could you make it more specific to tell it's the TCB version of the Certificate? maybe you can update the log 104, "Received certificate successfully, TCB version: XXX"